### DISCUSSION TITLE
Can't open a note from the "Note recall" pop-up window

### DISCUSSION THREAD

**Discussion by nerosun64** [coddingtonbear/obsidian-web#210] [Ideas]
### Describe the bug

I can't open the note from the "Note recall" pop-up window. I click on the link, but nothing happens. 
However, I can open the link to the same note using obsidian-url (`obsidian://Path%2FTo%20the%20note` or `obsidian://open?vault=Vault&file=Path%2FTo%20the%20note`)  in Google Chrome.
![image](https://github.com/user-attachments/assets/763db5ac-08ab-4d07-a3bd-2667126675cb)

I granted all the requested permissions on the settings page. I even manually enabled the option "Allow local files to be opened via links."
![image](https://github.com/user-attachments/assets/006fe630-dd4e-44c2-af16-605780101c9a)
![image](https://github.com/user-attachments/assets/eb5c8e10-e20b-4732-9712-79f7dabbd700)

![image](https://github.com/user-attachments/assets/b2231374-e2a3-4122-975d-f49083e727b5)

The web clipper itself works fine; notes are created.
The Local REST API works fine because notes from the web clipper are created without any issues.

Tested on main and freshly created pure chrome account.

Obsidian: 1.6.5
Obsidian web: 3.4.1
Local REST API: 2.4.3
Google Chrome: 126.0.6478.127

[obsidian-local-rest-api.bug_report.json](https://github.com/user-attachments/files/16203302/obsidian-local-rest-api.bug_report.json)

### To Reproduce

0. Enable all "Note recall" options, grant all permisions.
1. Create a note with the property 'url' or mention the URL in the note.
2. Go to this URL and wait for the Obsidian Web pop-up window to appear.
3. Click on the link that leads to the corresponding note.
4. Nothing happens (expected to open the note).

### Operating System

Windows 10

### Browser

Google Chrome

### Browser Version

126.0.6478.127

**Comment by nerosun64** [comment]
Okay, my bad. The note opens, but there's no signal about it. It just opens in the background on the currently active Obsidian tab. Is it possible to make the note open in a new tab at least, because something important might be open in the active tab?

_Moved from "bug report" to "ideas"._

**Comment by coddingtonbear** [comment]
Unfortunately, too my knowledge there is no API provided by Obsidian for focusing the window.  If you can find one, though, feel free to file a pull request!
